docs/src/rustc-hacks.md

```diff
@@ -33,6 +33,25 @@ You may also need to install the `rustc-dev` package using rustup
 rustup toolchain install nightly --component rustc-dev
 ```
 
+#### Debugging in VS code
+
+To debug Kani in VS code, first install the [CodeLLDB extension](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb).
+Then add the following lines at the top of the function `codegen_crate` in `kani-compiler/src/codegen_cprover_gotoc/compiler_interface.rs` (see [the CodeLLDB manual](https://github.com/vadimcn/vscode-lldb/blob/master/MANUAL.md#attaching-debugger-to-the-current-process-rust) for details):
```

Comment:
Does it mean that the debugger only works past this point? What happens if you add to the main function?

Reply:
Good point, I wasn't sure about the interaction with the driver, but you're right, adding at at the start of the main function is better. (I checked that it works too.)

Reply:
we have two binaries, the driver and the compiler. It will depend on which process you want to debug. I suggest leaving the doc generic.

Reply:
I've changed it in my latest push to just say "start of the main function".